src/progress_bar_exercise/components/hider/Hider.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from "react";
+
+/**
+ * Hider Component removes children components from the
```

Comment:
I didn't get enough time to implement this Hider class. In a normal PR, I would either make the call to block the work until it's implemented and the Progress Bar implements it, or block PR until it's removed. No dead code!